core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -290,4 +298,43 @@ private void onChannelTerminated() {
     // Release the transport factory so that it can deallocate any resources.
     transportFactory.release();
   }
+
+  private final TransportManager tm = new TransportManager() {
+    @Override
+    public void updateRetainedTransports(ServerAddress[] addrs) {
+      // TODO
+    }
+
+    @Override
+    public ListenableFuture<ClientTransport> getTransport(final ServerAddress addr) {
```

Comment:
As discussed offline, maybe change this (and above) to SocketAddress. That would end up removing all current usages of getName() (since it would use SocketAddress equality instead).